Formula/monero.rb

```diff
@@ -7,6 +7,11 @@ class Monero < Formula
   license "BSD-3-Clause"
   revision 1
 
+  livecheck do
+    url "https://www.getmonero.org/blog/tags/releases.html"
+    regex(/Monero GUI .*?v?(\d+(?:\.\d+)+)["' >]/i)
```

Comment:
```suggestion
    url :stable
    strategy :github_latest
```

Reply:
I also would recommend to use the Github stable tag. (I help with monero release engineering).

Reply:
Sounds good to me.